.circleci/config.yml

```diff
@@ -888,6 +891,248 @@ jobs:
                 slack_bot_token: $SLACK_BOT_TOKEN
                 slack_channel: "acf-ohs-ttahub--contractor-customer-team"
     resource_class: large
+  backup_upload:
+    executor: aws-executor
+    steps:
+      - run:
+          name: Install Dependencies
+          command: |
+            # update
+            sudo apt-get update
+            # Install PostgreSQL Client
+            sudo apt-get install -y postgresql-client
+            # Install jq
+            sudo apt-get install -y jq
+            # Install zip & unzip
+            sudo apt-get install -y zip unzip
+            # Install Curl
+            sudo apt-get install -y curl
+            # Install Cloud Foundry CLI
+            wget -q -O - https://packages.cloudfoundry.org/debian/cli.cloudfoundry.org.key | sudo apt-key add -
+            echo "deb https://packages.cloudfoundry.org/debian stable main" | sudo tee /etc/apt/sources.list.d/cloudfoundry-cli.list
+            sudo apt-get update
+            sudo apt-get install cf8-cli
+            # cleanup
+            rm -rf /var/lib/apt/lists/*
+
+      - run:
+          name: Connect to DB
+          command: |
+            # Create the script file
+            cat << 'EOF' > /tmp/setup_db_connection.sh
```

Comment:
I think you can use `\<<<` instead of `<<` so CircleCI's config parser thing doesn't think this is the beginning of a pipeline variable like `<<SOME_SECRET>>`